roomee-api/roomee-api/Controllers/UserController.cs

```diff
@@ -47,16 +46,63 @@ public class UserController : ControllerBase {
 				return Ok(JsonConvert.SerializeObject(user, Formatting.Indented));
 			}
 		}
+		
+		[HttpGet]
+		[Route("roomees/{roomId}")]
+		public IActionResult GetMyRoomees([FromRoute][Required] string roomId, [FromHeader][Required] string token) {
+			if (roomId == string.Empty || roomId == null) {
+				return Problem("RoomId cannot be empty");
+			}
```

Comment:
Why is room id defined as a string if it's an int in the database? Also because the [Required] tag is there, you won't need to check it with the if statement because the API will already know.